templates/shared/_credentials-exit-survey.html

```diff
@@ -0,0 +1,1004 @@
+                      <span class="p-radio__label" id="ExitSurveyShortFormDifficulty3">Too hard</span>
+                    </label>
+                  </div>
+                </div>
+              </fieldset>
+            </li>
+            <li class="p-list__item">
+              <label for="ExitSurveyShortFormQuestionTimeAllocated"
+                id="LblExitSurveyShortFormQuestionTimeAllocated">What do you think about the amount of
+                time allocated for the short-form questions?</label>
+              <fieldset id="ExitSurveyShortFormQuestionTimeAllocated" name="ExitSurveyShortFormQuestionTimeAllocated"
+                aria-labelledby="LblExitSurveyShortFormQuestionTimeAllocated InstructExitSurveyShortFormQuestionTimeAllocated">
+                <div class="row">
+                  <div class="col-3">
+                    <label class="p-radio--inline">
+                      <input type="radio" class="p-radio__input" name="ExitSurveyShortFormQuestionTimeAllocated"
+                        aria-labelledby="ExitSurveyShortFormQuestionTimeAllocated1" value="Too short">
+                      <span class="p-radio__label" id="ExitSurveyShortFormQuestionTimeAllocated1">Too short</span>
+                    </label>
+                  </div>
+                  <div class="col-3">
+                    <label class="p-radio--inline">
+                      <input type="radio" class="p-radio__input" name="ExitSurveyShortFormQuestionTimeAllocated"
+                        aria-labelledby="ExitSurveyShortFormQuestionTimeAllocated2" value="The right amount of time">
+                      <span class="p-radio__label" id="ExitSurveyShortFormQuestionTimeAllocated2">The right amount of
+                        time</span>
+                    </label>
+                  </div>
+                  <div class="col-3">
+                    <label class="p-radio--inline">
+                      <input type="radio" class="p-radio__input" name="ExitSurveyShortFormQuestionTimeAllocated"
+                        aria-labelledby="ExitSurveyShortFormQuestionTimeAllocated3" value="Too long">
+                      <span class="p-radio__label" id="ExitSurveyShortFormQuestionTimeAllocated3">Too long</span>
+                    </label>
+                  </div>
+                </div>
+              </fieldset>
+            </li>
+          </ul>
+          <h4>Lab Questions</h4>
+          <ul class="p-list">
+            <li class="p-list__item">
+              <label for="ExitSurveyRelevanceofLabQuestions" id="LblExitSurveyRelevanceofLabQuestions">This exam is a
+                prerequisite for more advanced levels of certification and certifies a fundamental level of knowledge in
+                Linux/OSS community, Ubuntu Desktop, and Ubuntu Server. How relevant are the <strong>Lab
+                  Questions</strong> in
+                relation to this goal. </label>
```

Comment:
Replace period with question mark.

Reply:
Replace "in the Linux/OSS community, Ubuntu Desktop, and Ubuntu Server" with "in vendor-neutral Linux as specified in our syllabus."